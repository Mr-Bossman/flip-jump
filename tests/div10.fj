startup

loop:
print_hex_uint w, var, 1
output ','
bit.div10 w, dst, var
dec2ascii ascii, var
print ascii
output '\n'
bit.mov w, var, dst
bit.if1 w, var, loop
loop


var: bit.vec w, 1281 // (128, 12, 1)
dst: bit.vec w
ascii: bit.vec 8
