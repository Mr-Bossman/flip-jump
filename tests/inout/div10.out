0x501,1
0x80,8
0xC,2
0x1,1
